ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py

```diff
@@ -51,7 +51,7 @@ class OMetaPatchMixin(Generic[T]):
     client: REST
 
     def _validate_instance_description(
-        self, entity: Type[T], entity_id: Union[str, basic.Uuid], force: bool = False
```

Comment:
Was `force` unused?

Reply:
yes, looks like not used in the code anywhere..

Reply:
yeah, the linter was crying about that